articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-users.md

```diff
@@ -304,6 +342,12 @@ Universal Windows Platform (UWP) projects, but using the **UWP** project (with n
        {
            base.OnActivated(args);
 
+            if (args.Kind == ActivationKind.Protocol)
+            {
+                ProtocolActivatedEventArgs protocolArgs = args as ProtocolActivatedEventArgs;
+                TodoItemManager.DefaultManager.CurrentClient.ResumeWithURL(protocolArgs.Uri);
+            }
+
```

Comment:
Remove the next 6 lines of code. They are replaced by the new `ResumeWithURL` code you added above.